helm/kiam/CHANGELOG.md

```diff
@@ -1,5 +1,42 @@
 # Helm Chart Changelog
 
+## 5.9.0
+REPLACEME 2020
```

Comment:
this all looks good thanks, just need to remove this line

Reply:
Awesome, thanks for looking! 

The idea was that we'd replace that when merging :)
If you want I can change it now or you can change as you merge.